src/main/java/model/Colors.java

```diff
@@ -0,0 +1,11 @@
+package model;
+
+public enum Colors {
+    Red,
+    Orange,
+    Yellow,
+    Green,
+    Blue,
+    Purple,
+    Pink

```

Comment:
According to the Java style guide, the names of constants (including enum values) should be in uppercase letters with underscores to separate words. Please update the enum values to follow this convention.